lib/challenge.js

```diff
@@ -60,6 +86,20 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function getPetInfo(name, species, breed, noise) = {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise
+  this.makeNoise = makeNoise();
```

Comment:
This line doesn't mean what you think it means. It's creating a `makeNoise` attribute for all pets, and setting the value to the result of some function that exists elsewhere called `makeNoise`.